requirements/dev.txt

```diff
@@ -4,173 +4,229 @@
 #
 #    pip-compile --output-file=requirements/dev.txt requirements/dev.in
 #
--e .
+-e git+https://github.com/kytos-ng/kytos.git#egg=kytos
     # via -r requirements/dev.in
--e git+https://github.com/kytos-ng/sphinx-theme.git#egg=kytos-sphinx-theme
+-e file:///home/alopalao/Desktop/up/of_multi_table
```

Comment:
```suggestion
-e .
```

This line here always need to be updated after regenerating. Can you update it? 

btw, I linked scrutinizer with this repo, so now on every commit and PRs you'll also be able to see `tox` execution on scrutinizer 

https://scrutinizer-ci.com/g/kytos-ng/of_multi_table/inspections/8f1a491e-5cb1-4ec3-9804-3c35b087a9f6/log

Reply:
Got it